src/xunit.performance.api/Profilers/Etw/DotNetModule.cs

```diff
@@ -8,29 +8,29 @@ namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
     /// <summary>
     /// Loaded .NET module for the corresponding process.
     /// </summary>
-    internal sealed class EtwDotNetModule : EtwModule
+    internal sealed class DotNetModule : Module
     {
         /// <summary>
-        /// Initializes a new instance of the EtwDotNetModule class.
+        /// Initializes a new instance of the DotNetModule class.
         /// </summary>
         /// <param name="fullName"></param>
         /// <param name="checksum"></param>
         /// <param name="id"></param>
-        public EtwDotNetModule(string fullName, int checksum, long id)
+        public DotNetModule(string fullName, int checksum, long id)
            : base(fullName, checksum)
         {
             Id = id;
-            Methods = new List<EtwDotNetMethod>();
+            Methods = new List<DotNetMethod>();
         }
 
         /// <summary>
-        /// Module Id
+        /// Module Id.
         /// </summary>
         public long Id { get; }
 
         /// <summary>
         /// Collection of loaded methods for this module.
         /// </summary>
-        internal IList<EtwDotNetMethod> Methods { get; }
+        internal IList<DotNetMethod> Methods { get; }
```

Comment:
Do we ever want to expose per-method counts?  I'm wondering if because we're sampling if this will be too coarse to want to use in a benchmarking system.

Reply:
Yes, I could expose it. I've been very conservative on the things I make public, because I'm working around the needs of the CoreClr performance scenarios only, but still designing a reusable interface that makes sense for most cases.

---
In reply to: [146059550](https://github.com/Microsoft/xunit-performance/pull/242#discussion_r146059550) [](ancestors = 146059550)

Reply:
I was thinking that we could also collect "number of jitted methods" as a metric (I already have that information).

---
In reply to: [146065630](https://github.com/Microsoft/xunit-performance/pull/242#discussion_r146065630) [](ancestors = 146065630,146059550)